src/jit/codegencommon.cpp

```diff
@@ -10317,10 +10333,16 @@ void CodeGen::genFuncletProlog(BasicBlock* block)
 
     compiler->unwindBegProlog();
 
-    // TODO Save callee-saved registers
+    inst_RV(INS_push, REG_FPBASE, TYP_REF);
```

Comment:
@jkotas In the changes to UnwindEbpDoubleAlignFrame(), there is a call to SetEbpLocation() that appears to point to this saved EBP. Is that not required? (Or maybe I'm reading it wrong?)